libbeat/beat/beat.go

```diff
@@ -87,9 +91,7 @@ type Creator func(*Beat, *common.Config) (Beater, error)
 // Beat contains the basic beat data and the publisher client used to publish
 // events.
 type Beat struct {
-	Name      string              // Beat name.
-	Version   string              // Beat version number. Defaults to the libbeat version when an implementation does not set a version.
-	UUID      uuid.UUID           // ID assigned to a Beat instance.
+	Info      publisher.BeatInfo  // beat metadata.
```

Comment:
I can also make use of this in https://github.com/elastic/beats/pull/3527 Perhaps we should put it into a separate package to prevent circular imports.